src/index.scss

```diff
@@ -0,0 +1,5 @@
+body {
+    div#root{
```

Comment:
@e-ian , please use indentation of two spaces here, and everywhere else in the app.

Reply:
I have set indentation to two spaces in all files